docs/TypeScript.md

```diff
@@ -555,7 +555,7 @@ An example can be found in the rules_nodejs repo under `packages/typescript/test
 &gt; To apply the fix, you can use the patch-package package to apply this patch:
 &gt; https://gist.github.com/alexeagle/ba44b2601bd7c953d29c6e8ec44d1ef9
 
-Defaults to `None`
+Defaults to `Label("@npm//typescript/bin:tsc")`
```

Comment:
Was this true before but the docs were wrong?

`_DEFAULT_TSC` had the `@npm` stripped though?

Reply:
previously the default was in the rule implementation, but we run stardoc against the wrapper macro, so it wasn't seen. Now the default is in the macro so it appears here.

The published package uses the `#BEGIN/END-INTERNAL` fencing to strip the `@npm` prefix, but stardoc runs against the sources so that stripping isn't visible to it.

Either way I think the documentation isn't very useful about what the default is.